index.js

```diff
@@ -15,4 +15,8 @@ if (typeof window !== 'undefined') {
   require('classlist-polyfill'); // eslint-disable-line global-require
 }
 
+// Polyfill window.matchMedia (primarily for IE9)
+require('matchmedia-polyfill');
```

Comment:
these need to be in the `if window` block above, so that they don't fail in node